packages/files/file.pony

```diff
@@ -412,81 +533,93 @@ class File
     let pos = position()
     _seek(0, 2)
     let len = position()
-    _seek(pos.isize(), 0)
+    _seek(pos.i64(), 0)
     len
 
-  fun ref seek_start(offset: USize) =>
+  fun ref seek_start(offset: USize): File =>
```

Comment:
Why return `this`? We don't need method chaining now that we have the chaining operator `.>`.